File: Java. Руководство для начинающих Шилдт/6/MyClass.java
class MyClass{
	int x;
	MyClass(){
		System.out.println("Внутри MyClass().");
		x=0;
	}
	MyClass(int i){
		System.out.println("Внутри MyClass(int).");
		x=i;
	}
	MyClass(double d){
		System.out.println("Внутри MyClass(double).");
		x=(int)d;
	}
	MyClass(int i,int j){
		System.out.println("Внутри MyClass(int,int).");
		x=i*j;
	}
}

File: KotlinLessons/KotlinCoroutine/p0311testing/main/java/com/korobeynikov/p0311testing/MyClass.kt
package com.korobeynikov.p0311testing

import androidx.lifecycle.ViewModel
import androidx.lifecycle.viewModelScope
import kotlinx.coroutines.CoroutineScope
import kotlinx.coroutines.coroutineScope
import kotlinx.coroutines.delay
import kotlinx.coroutines.launch

class MyClass: ViewModel() {
    suspend fun someMethod():String{
        return coroutineScope{
            println("before launch")
            launch{
                println("launch")
            }
            println("after launch")
            "abc"
        }
    }
}